js/web/_i18n/it.json

```diff
@@ -71,239 +71,239 @@
-		"Boxes.Productions.Headings.decoration": "Dekorations",
-		"Boxes.Productions.Headings.earning": "Guadagnare",
-		"Boxes.Productions.Headings.goods": "Edifici merci",
-		"Boxes.Productions.Headings.greatbuilding": "Edifici leggendari",
+		"Boxes.Productions.Headings.decoration": "Decorazioni",
+		"Boxes.Productions.Headings.earning": "Guadagno",
+		"Boxes.Productions.Headings.goods": "Edifici di beni",
+		"Boxes.Productions.Headings.greatbuilding": "Grandi Edifici",
 		"Boxes.Productions.Headings.main_building": "Municipio",
-		"Boxes.Productions.Headings.name": "Name", //Todo: Translate
+		"Boxes.Productions.Headings.name": "Nome",
 		"Boxes.Productions.Headings.number": "Numero",
 		"Boxes.Productions.Headings.production": "Edificio di produzione",
 		"Boxes.Productions.Headings.random_production": "Produzioni casuali",
 		"Boxes.Productions.Headings.residential": "Edifici residenziali",
-		"Boxes.Productions.Headings.street": "Streets",
+		"Boxes.Productions.Headings.street": "Strade",
 		"Boxes.Productions.ModeGroups": "Gruppi",
-		"Boxes.Productions.ModeSingle": "Single",
+		"Boxes.Productions.ModeSingle": "Singolo",
 		"Boxes.Productions.SearchInput": "Cerca edificio ...",
 		"Boxes.Productions.Title": "Panoramica della produzione",
-		"Boxes.Productions.Total": "Total:",
+		"Boxes.Productions.Total": "Totale:",
 		"Boxes.Settings.Active": "Attivo",
 		"Boxes.Settings.Inactive": "Inattivo",
 		"Boxes.Settings.Title": "Impostazioni",
 		"Boxes.StrategyPoints.BuyableFP": "Acquistabile:",
-		"Boxes.StrategyPoints.FPBar": "FP-Stock:",
+		"Boxes.StrategyPoints.FPBar": "Magazzino PF:",
 		"Boxes.Technologies.DescInStock": "Disponibile",
 		"Boxes.Technologies.DescRequired": "Richiesto",
 		"Boxes.Technologies.DescStillMissing": "<span style='color:#29b206'>Eccesso</span> / <span style='color:#ef1616'>Manca</span>",
 		"Boxes.Technologies.NoTechs": "Hai raggiunto la fine di questa era",
 		"Boxes.Technologies.Resource": "Risorsa",
 		"Boxes.Technologies.Title": "Costo della ricerca per",
-		"Boxes.Units.Attack": "Attack",
-		"Boxes.Units.Bind": "Bound",
+		"Boxes.Units.Attack": "Attacco",
+		"Boxes.Units.Bind": "Caserma",
 		"Boxes.Units.Defend": "Difesa",
-		"Boxes.Units.NextUnitsIn": "Le prossime unità __count__ arriveranno in <span class = \"alca-countdown \"> </span> a __harvest__ Uhr",
-		"Boxes.Units.NotFilled": "non riempito",
+		"Boxes.Units.NextUnitsIn": "Le prossime unità __count__ arriveranno in <span class = \"alca-countdown \"> </span> alle ore __harvest__",
+		"Boxes.Units.NotFilled": "Non riempito",
 		"Boxes.Units.Proportionally": "Proportionally",
 		"Boxes.Units.Quantity": "Quantità",
 		"Boxes.Units.ReadyToLoot": "Pronto a saccheggiare!",
 		"Boxes.Units.Status": "Status",
 		"Boxes.Units.Title": "Panoramica dell'esercito",
-		"Boxes.Units.Unbind": "Unbound",
+		"Boxes.Units.Unbind": "Indipendenti",
 		"Boxes.Units.Unit": "Unità",
-		"DateTime": "D / M / YY h: mm: ss a",
```

Comment:
Dont Change this, revert it to: "D / M / YY h: mm: ss a",

Reply:
noticed it, I forgot the data wouldn't have changed